02/app.js

```diff
@@ -1,2 +1,23 @@
 const x = 5;
-let result = 0;
\ No newline at end of file
+let result = 0;
+for(let i=1; i<=x; i++) {
+    result = result + i;
+    console.log;
+}
+console.log('Wynik to' + result);
+prompt('wynik to' + result);
```

Comment:
`prompt` prosi o wpisanie jakiś danych, jeśli chcesz tylko wyświetlić to możesz użyć `alert`